etk/core.py

```diff
@@ -546,16 +582,28 @@ def _relevant_text_from_context(text_or_tokens, results, field_name):
                             new_end = tokens_len
                         else:
                             new_end = end + 5
+
+                        # result['context']['text'] = ' '.join(
+                        #     text_or_tokens[new_start:new_end])
+                        # result['context']['tokens_left'] = text_or_tokens[
+                        #     new_start:start]
+                        # result['context'][
+                        #     'tokens_right'] = text_or_tokens[end:new_end]
+
```

Comment:
why is this commented out code here? Makes me wonder how are you guys resolving conflicts?